smartseq2_single_sample/run_multi_samples/scripts/calculate_cost_workflow.py

```diff
@@ -0,0 +1,253 @@
+import argparse
+import json
+import math
+import urllib.request, urllib.error, urllib.parse
+import dateutil.parser
+from io import StringIO
+import gzip
+import requests
+
+GCE_MACHINE_TYPES_URL = "http://cloudpricingcalculator.appspot.com/static/data/pricelist.json"
+TOTAL_WORKFLOW_COST = 0
+
+# load the US `pricing for both persistent disk and compute engine
+def get_gce_pricing():
+    response = urllib.request.urlopen(GCE_MACHINE_TYPES_URL)
+    data = response.read()
+
+    if response.info().get('Content-Encoding') == 'gzip':
+        buf = StringIO(data)
+        f = gzip.GzipFile(fileobj=buf)
+        data = f.read()
+
+    pricing = json.loads(data)
+
+    data = {}
+    for k, v in pricing.items():
+        if k == "gcp_price_list":
+            for k2, v2 in v.items():
+                if k2.startswith("CP-COMPUTEENGINE-VMIMAGE"):
+                    data[k2.replace("CP-COMPUTEENGINE-VMIMAGE-", "").lower()] = v2['us']
+                if k2.startswith("CP-COMPUTEENGINE-STORAGE-PD"):
```

Comment:
Since the last two if clauses contain the same code, you can combine them into `if k2.startswith("CP-COMPUTEENGINE-STORAGE-PD") or k2.startswith("CP-COMPUTEENGINE-STORAGE-PD-CAPACITY")`

Reply:
at least I can change this one. :)